react/Minicart.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useCssHandles } from 'vtex.css-handles'
+
+import MinicartIconButton from './components/MinicartIconButton'
+import DrawerMode from './components/DrawerMode'
+
+import { MinicartContextProvider, useMinicartState } from './MinicartContext'
+import PopupMode from './components/Popup'
+
+const CSS_HANDLES = ['minicartWrapperContainer', 'minicartContainer'] as const
+
+const Minicart: FC<MinicartProps> = ({
+  maxDrawerWidth = 400,
+  drawerSlideDirection = 'rightToLeft',
+  linkVariationUrl = '/checkout/#/cart',
```

Comment:
(Later we should do what the checkout team PR does of checking which version of the checkout app is installed)

Reply:
Yeah! I'll implement that in the next Pull Request :)